RunningOrder/Sprint/Models/Sprint.swift

```diff
@@ -23,3 +23,40 @@ extension Sprint {
 
 extension Sprint: Equatable { }
 extension Sprint: Hashable { }
+
+struct SearchSection: Identifiable {
+    var id: UUID { return UUID() }
+    var name: String
+    var items: [SearchItem]
+}
+
+extension SearchSection {
+    enum SectionType: String {
+        case story // Jira reference + story's name
+        case epic // epic name
+        case people // story's creator name
+
+        var iconName: String {
+            switch self {
+            case .story:
+                return "list.bullet.rectangle"
+            case .epic:
+                return "folder.fill"
+            case .people:
+                return "person.circle"
+            }
+        }
+    }
+}
+
+struct SearchItem: Identifiable, Hashable {
+    static func == (lhs: SearchItem, rhs: SearchItem) -> Bool {
+        lhs.name == rhs.name
+    }
+
+    var id: UUID { return UUID() }
```

Comment:
same : add conformance in an extension, not with a computed if you create a new not fixed value in it

you made SeachItem Equatable by name, why not using name in the id ?